src/app/components/merge/OptionsDropdown.js

```diff
@@ -3,10 +3,10 @@ import PropTypes from 'prop-types';
 import { c } from 'ttag';
 import { SimpleDropdown, DropdownMenu, DropdownButton } from 'react-components';
 
-const OptionsDropdown = ({ contactID, index, canDelete, onClickDetails, onClickDelete, onClickUndelete }) => {
-    const color = canDelete ? 'color-global-warning' : 'color-pv-green';
+const OptionsDropdown = ({ contactID, canDelete, onClickDetails, onClickDelete, onClickUndelete }) => {
+    const color = canDelete ? 'pm-button--redborder' : 'pv-button-greenborder';
```

Comment:
Did you check with Nico about this?

Reply:
Good that you point it out since actually `OptionsDropdown` is not used anymore. I forgot to delete it! :sweat_smile: